fixtures/packages/iron-icon/expected/.travis.yml

```diff
@@ -22,8 +21,8 @@ addons:
     packages:
       - google-chrome-stable
 script:
-  - xvfb-run polymer test
+  - xvfb-run polymer-test --npm
```

Comment:
why is there a dash in `polymer-test` now?

Reply:
Oops, my replace is wrong